mlflow/deployments/plugin_manager.py

```diff
@@ -96,7 +99,7 @@ def __getitem__(self, item):
             )
             raise MlflowException(msg, error_code=RESOURCE_DOES_NOT_EXIST)
 
-        if isinstance(plugin_like, entrypoints.EntryPoint):
+        if isinstance(plugin_like, (importlib_metadata.EntryPoint, importlib.metadata.EntryPoint)):
```

Comment:
After `importlib_metadata` is imported, `importlib.metadata.entry_points` returns a list of `importlib_metadata.EntryPoint` instances, which is weird.